client/aci/types.go

```diff
@@ -65,6 +65,21 @@ type Container struct {
 	ContainerProperties `json:"properties,omitempty"`
 }
 
+// InitContainerDefinition is a initContainer instance.
+type InitContainerDefinition struct {
+	Name                    string `json:"name,omitempty"`
+	InitContainerProperties `json:"properties,omitempty"`
+}
+
+// InitContainerProperties is the initContainer instance properties.
+type InitContainerProperties struct {
+	Image                string                          `json:"image,omitempty"`
+	Command              []string                        `json:"command,omitempty"`
+	EnvironmentVariables []EnvironmentVariable           `json:"environmentVariables,omitempty"`
+	InstanceView         ContainerPropertiesInstanceView `json:"instanceView,omitempty"`
+	VolumeMounts         []VolumeMount                   `json:"volumeMounts,omitempty"`
+}
+
```

Comment:
We don't need that anymore, you can use the SDK [models](https://github.com/Azure/azure-sdk-for-go/blob/main/services/containerinstance/mgmt/2021-10-01/containerinstance/models.go#L874) now.

Reply:
removed